operator/operator.make

```diff
@@ -52,17 +53,19 @@ OPERATOR_OUTPUT_DIR := $(CATALOG_DIR)/$(MAJOR_MINOR_PATCH_VERSION)
 
 # Generate CRD and add kustomization support
 operator-generate-crd: controller-gen
-	@echo "Generating CRD in $(MANIFESTS_DIR)/crd ..."
-	$(CONTROLLER_GEN) crd:trivialVersions=true,crdVersions=v1beta1 paths=./pkg/apis/... output:dir=$(MANIFESTS_DIR)/crd/
-	@echo "resources: [pmem-csi.intel.com_deployments.yaml]" > $(MANIFESTS_DIR)/crd/kustomization.yaml
+	@echo "Generating CRD in $(CRD_DIR) ..."
+	$(CONTROLLER_GEN) crd:trivialVersions=true,crdVersions=v1beta1 paths=./pkg/apis/... output:dir=$(CRD_DIR)/
+	version="$(shell $(CONTROLLER_GEN) --version |cut -f2 -d':')"; \
+	for f in $(shell ls $(CRD_DIR)); do \
```

Comment:
That looks complicated. Why not just `for f in $(CRD_DIR)/*; do`?

Reply:
> That looks complicated. Why not just `for f in $(CRD_DIR)/*; do`?

Yes, I should have done this. Changed it now.